lib/Ohmbrewer_Rhizome.h

```diff
@@ -247,6 +253,17 @@ namespace Ohmbrewer {
          * @return Error or success code, according to the requirements specified by addSprout
          */
         int addRIMS(char* params);
+
+        /**
+         * Adds a sprout to the list and rebuids the index
+         * @param equipment The sprout being added
+         */
+        void saveNewSprout(Equipment* sprout);
```

Comment:
Mostly to minimize synonyms. This method does the same thing as the existing addSprout method, except the existing one includes the extra handling necessary for Strings. Both methods just provide the necessary machinations to put a particular type of object in the Sprouts list. As I see it, this is just the sort of use case that [method overloading](http://www.tutorialspoint.com/cplusplus/cpp_overloading.htm) was designed for.